lib/api/api.go

```diff
@@ -187,6 +187,16 @@ func (s *service) getListener(guiCfg config.GUIConfiguration) (net.Listener, err
 		return nil, err
 	}
 
+	if guiCfg.Network() == "unix" && guiCfg.SocketPermissions() != 0 {
+		// We should error if this fails under the assumption that these permissions are
+		// required for operation. If the configuration option isn't correctly formatted,
+		// then we will ignore it.
```

Comment:
I don't see anything about formatting here, I guess that happens within `.SocketPermissions()`? In that case move the second sentence there (or just drop it).

Reply:
Done.